Makefile

```diff
@@ -2,8 +2,8 @@
 
 SHELL = /bin/bash
 
-MAKEGITHUBACCOUNT = irods
-MAKEIRODSVERSION = main
+GITHUB_ACCOUNT = irods
+GIT_COMMITTISH = main
```

Comment:
Should this be `GIT_COMMITISH`? Never seen it with two t's before.

Reply:
Oops, should have looked it up and this is the way to go. Looks good

https://git-scm.com/docs/gitglossary